### PR TITLE
Fleshed out layout based on UIKit

### PR DESCRIPTION
**Pull request by nathf**

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -1 +1,3 @@
 <%= react_component("App", props: @default_props) %>
+
+<a href="/transitions" role="button">Start Transition <i class="fa fa-chevron-right" aria-hidden="true"></i></a>
```
Comment (micapam) [inline app/views/home/index.html.erb]: Shouldn't this be 'list transitions' ?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,17 +1,31 @@
...
+    <title>GOV.AU Transitions</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <%= csrf_meta_tags %>
+    <%= favicon_link_tag %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.16/webfont.js"></script>
+    <script>
+      WebFont.load({
```
Comment (micapam) [inline app/views/layouts/application.html.erb]: Why is this stuff inline, and synchronous? Couldn't we [install it via npm](https://github.com/typekit/webfontloader) and load it asynchronously?

File: app/views/partials/_header.html.erb [hunk 3]
```diff
@@ -6,17 +12,17 @@
 <header role="banner">
   <section class="prerelease-govau--header">
     <div class="wrapper">
-      <div class="govau--logo">
-        <!-- <img src="{% asset_path logo-govau-full-2x_alpha.png %}" alt="gov.au crest logo" width="270"> -->
 
+      <a class="govau--logo" href="/">
```
Comment (micapam) [inline app/views/partials/_header.html.erb]: Shouldn't this link off to global govau homepage?

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (alecky) [inline app/views/layouts/application.html.erb]: This also causes the font flash on the page. this was a temp fix from the ui kit people

File: app/views/partials/_header.html.erb [hunk 4]
```diff
@@ -1,3 +1,9 @@
+<!--[if lt IE 8]>
```
Comment (alecky) [inline app/views/partials/_header.html.erb]: Should this be less than 9??

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (nathf) [inline app/views/layouts/application.html.erb]: @micapam fair point, will move it to npm

**Comment by micapam** [comment]
LGTM

**Pull request merged**